workflow-testing/src/main/java/com/squareup/workflow1/testing/RenderTestResult.kt

```diff
@@ -38,10 +38,59 @@ public interface RenderTestResult<PropsT, StateT, OutputT, RenderingT> {
    * be useful if your actions are anonymous. If they are a sealed class or enum, use [verifyAction]
    * instead and write separate unit tests for your action implementations.
    */
+  @Deprecated(
```

Comment:
Not sure we need to deprecate this but maybe update the kdoc to say

"... invokes [block] with the resulting state and output. Note that the output is wrapped in [WorkflowOutput] to differentiate from the case of no Output emitted (`output == null`) or `null` emitted (`WorkflowOutput(null)`)."